framework/generated/generated_vulkan_enum_to_json.h

```diff
@@ -508,7 +510,6 @@ void FieldToJson(nlohmann::ordered_json& jdata, const VkVideoDecodeUsageFlagBits
 void FieldToJson(nlohmann::ordered_json& jdata, const VkVideoEncodeCapabilityFlagBitsKHR& value, const util::JsonOptions& options = util::JsonOptions());
 void FieldToJson(nlohmann::ordered_json& jdata, const VkVideoEncodeContentFlagBitsKHR& value, const util::JsonOptions& options = util::JsonOptions());
 void FieldToJson(nlohmann::ordered_json& jdata, const VkVideoEncodeFeedbackFlagBitsKHR& value, const util::JsonOptions& options = util::JsonOptions());
-void FieldToJson(nlohmann::ordered_json& jdata, const VkVideoEncodeFlagBitsKHR& value, const util::JsonOptions& options = util::JsonOptions());
```

Comment:
Why does VkVideoEncodeFlagBitsKHR get removed? Is it a duplicate entry?

Reply:
For some reason, the `requires` field of `VkVideoEncodeFlagsKHR` was removed from `vk.xml`.
```
-        <type requires="VkVideoEncodeFlagBitsKHR"                   category="bitmask">typedef <type>VkFlags</type> <name>VkVideoEncodeFlagsKHR</name>;</type>
+        <type                                                       category="bitmask">typedef <type>VkFlags</type> <name>VkVideoEncodeFlagsKHR</name>;</type>
```

Perhaps that's causing the generator to miss it somehow? Trying to figure it out.

Reply:
https://gitlab.khronos.org/vulkan/vulkan/-/merge_requests/6781

I think the requires was removed because nobody used the bitflag. So I believe my concern is misplaced. Yes the codegen removed it but that is expected. When the bitflag becomes used the codegen will re-add it.